src/main/common/maths.h

```diff
@@ -90,8 +90,8 @@ typedef union {
 
 int gcd(int num, int denom);
 float powerf(float base, int exp);
-int32_t applyDeadband(int32_t value, int32_t deadband);
-float fapplyDeadband(float value, float deadband);
+int32_t applyDeadband(const int32_t value, const int32_t deadband);
```

Comment:
`const` on arguments passed by value is useless in declarations
Only makes sense in definitions

Reply:
Ok. I assumed the compiler would complain if they did not match.

Reply:
They aren't even checked, all will be good :)